articles/logic-apps/logic-apps-limits-and-config.md

```diff
@@ -120,6 +120,21 @@ Following are limits for artifacts added to integration Account
 |Runtime endpoint tracking calls per 5 minutes |45,000|Can distribute workload across multiple accounts as needed|
 |Runtime endpoint blocking concurrent calls |~1,000|Decrease number of concurrent requests or reduce the duration as needed|
 
+Following are limits for count of artifacts added to Integration Account.
```

Comment:
"Following are the limits on the number of artifacts that you can add to an integration account."